armi/materials/lithium.py

```diff
@@ -43,8 +60,8 @@ def density(self, Tk=None, Tc=None):
         return 0.512  # g/cc
 
     def setDefaultMassFracs(self):
-        self.setMassFrac("LI6", 0.075)
-        self.setMassFrac("LI7", 0.925)
+        self.setMassFrac("LI6", nb.byName["LI6"].abundance)
```

Comment:
How do we know this "byName" dict won't be empty?

Sorry if that's an obvious question, it's the first time I've seen this dictionary, and by default, it starts empty:

https://github.com/terrapower/armi/blob/master/armi/nucDirectory/nuclideBases.py#L118

Reply:
When you import the `nuclideBases` module, the dict is populated. 

To be honest, the logic inside of `nuclideBases` that makes this happen is a little confusing to me, but it definitely happens when the module is initialized, based on my testing.

Reply:
Ack. I see, it's during the top-level project bootstrap:

https://github.com/terrapower/armi/blob/276c48fc11b582869dd1f7894c9fd989a0702e1a/armi/_bootstrap.py#L69

Well, that's not my favorite thing. But that's not your fault.

Thanks!